docs/FAQ.md

```diff
@@ -16,3 +16,11 @@ TypeError: compilation.mainTemplate.applyPluginsWaterfall is not a function
 ```
 
 - Fix this by bumping the `html-webpack-plugin` version to be `^3.2.0`
+
+##  I am using toolkit v3/webpack v4 and want to use the  exract-text-plugin, but am getting the error:
```

Comment:
I'm bad at typing. `extract-text-plugin`
And do we want to bullet the "Fix this by using...."?